pkg/util/helper/policy.go

```diff
@@ -25,6 +25,11 @@ func SetDefaultSpreadConstraints(spreadConstraints []policyv1alpha1.SpreadConstr
 			klog.Infof("Setting default MinGroups to 1")
 			spreadConstraints[i].MinGroups = 1
 		}
+
+		if spreadConstraints[i].MaxGroups == 0 && spreadConstraints[i].MinGroups > 0 {
```

Comment:
> 

If minGroups is less than zero? This part of the judgment is necessary.

Reply:
Isn't this a situation that has been intercepted by validate?

Reply:
> intercepted

No, mutating webhook works first!

Reply:
I recommend using `+kubebuilder:validation:Minimum=1`.

Reply:
> No, mutating webhook works first!

Ok

> I recommend using `+kubebuilder:validation:Minimum=1`.

Does this conflict with `// +optional`?

Reply:
> I recommend using `+kubebuilder:validation:Minimum=1` and `+kubebuilder:default=1`.

I don't see any difference in performance, because the webhook will always be called, but this part of the verification is done by the apiserver.